<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMovie</class>
 <widget class="QMainWindow" name="NewMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>فیلم جدید</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#NewMovie{
	background-color: #f2f2f2;
	border: none;
}
#centralwidget{
	border: none;
}
#section_center{
    background-color: white;
	border-radius: 12px;
	margin-top: 15px;
}

#title{
	color: #182848;
}

#icon_wrapper{
	background-color: #182848;
	border-radius: 50px;
}
#icon{
    border-radius: 50px;
}

#form{
	background-color: ;
}
#name_wrapper, #director_wrapper, #genre_wrapper, #ticket_wrapper{
	background-color: ;
}
#name_input, #director_input, #genre_input, #desc_input, #ticket_input, #cast_input{
	height: 45px;
	border: 1px solid #d6d6d6;	
	outline: none;
	background-color: ;
	color: #182848;
	border-radius: 6px;
	text-align: right;
}
#name_label, #director_label, #genre_label, #desc_label, #ticket_label, #cast_label{
	color: #818181;
}
#submit_btn{
	border: none;
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:1 #182848, stop: 0.2 #4b6cb7, stop:1 #182848);
	border-radius: 10px;
	color: white;
}
#submit_btn:hover{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:1 #4b6cb7, stop: 0.2 #182848, stop:1 #4b6cb7);
}

#footer_span{
	color: rgb(105, 105, 105);
	margin-top: 2px;
	margin-left: -5px;
}
#footer_log_btn, #footer_sign_btn{
	outline: none;
    border: none;
    color: rgb(59, 59, 59);
}
#footer_log_btn:hover, #footer_sign_btn:hover{
	color: #4b6cb7;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>818</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="section_center">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>800</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="header">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>25</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="header_wrapper">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="title">
                 <property name="font">
                  <font>
                   <family>B Yekan</family>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>فیلم جدید</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="icon_box" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QHBoxLayout" name="_3">
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QWidget" name="icon_wrapper" native="true">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="_2">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="icon">
                   <property name="minimumSize">
                    <size>
                     <width>90</width>
                     <height>85</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>90</width>
                     <height>85</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="Resources.qrc">:/images/iron-man-icon.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="form" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QWidget" name="name_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="leftMargin">
                  <number>9</number>
                 </property>
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>9</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="name_input">
                   <property name="maximumSize">
                    <size>
                     <width>320</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="name_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>نام فیلم</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="director_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="director_input">
                   <property name="maximumSize">
                    <size>
                     <width>320</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="director_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>کارگردان</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="cast_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <property name="leftMargin">
                  <number>9</number>
                 </property>
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>9</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="cast_input">
                   <property name="maximumSize">
                    <size>
                     <width>320</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="cast_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>بازیگران</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="genre_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QComboBox" name="genre_input">
                   <property name="minimumSize">
                    <size>
                     <width>320</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <item>
                    <property name="text">
                     <string>ترسناک</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>اکشن</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>علمی - تخیلی</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>درام</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>رمان</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>کمدی</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>موزیکال</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>زندگی نامه</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="genre_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>ژانر</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="desc_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QTextEdit" name="desc_input">
                   <property name="maximumSize">
                    <size>
                     <width>320</width>
                     <height>90</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="placeholderText">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="desc_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>توضیحات</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="ticket_wrapper" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="leftMargin">
                  <number>9</number>
                 </property>
                 <property name="topMargin">
                  <number>20</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>9</number>
                 </property>
                 <item>
                  <widget class="QSpinBox" name="ticket_input">
                   <property name="minimumSize">
                    <size>
                     <width>320</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>320</width>
                     <height>90</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="ticket_label">
                   <property name="font">
                    <font>
                     <family>B Yekan</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>تعداد بلیط</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="btn_wrapper" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QPushButton" name="submit_btn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>B Yekan</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>ثبت فیلم</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
